src/hooks/AuditStatus/useLoadEntityList.ts

```diff
@@ -86,6 +87,11 @@ const useLoadEntityList = ({
     });
   };
 
+  const { data: reportTaskUuid } = useGetV2TasksIDId(
+    { pathParams: { id: entity?.taskId! } },
+    { enabled: isProjectReport && !!entity?.taskId }
+  );
```

Comment:
Same comment here on including the task UUID in the response instead of requiring another API request.